packages/access-api/src/service/index.js

```diff
@@ -1,14 +1,14 @@
 import * as Server from '@ucanto/server'
 import { Failure } from '@ucanto/server'
-import * as Space from '@web3-storage/access/capabilities/space'
+import * as Space from '@web3-storage/capabilities/space'
 import { voucherClaimProvider } from './voucher-claim.js'
 import { voucherRedeemProvider } from './voucher-redeem.js'
 import * as DID from '@ipld/dag-ucan/did'
 import {
   delegationToString,
   stringToDelegation,
 } from '@web3-storage/access/encoding'
-import { top } from '@web3-storage/access/capabilities/top'
+import { top } from '@web3-storage/capabilities/top'
```

Comment:
Nit: I would reexport `top` from each capability group, so you could simply use `Space.top` if needed as opposed to needing to import second module everywhere.

Reply:
did it